sqlmesh/migrations/v0019_add_env_suffix_target.py

```diff
@@ -0,0 +1,61 @@
+"""Add support for environment suffix target."""
+import pandas as pd
+from sqlglot import exp
+
+from sqlmesh.utils.migration import index_text_type
+
+
+def migrate(state_sync):  # type: ignore
+    engine_adapter = state_sync.engine_adapter
+    environments_table = "_environments"
+    if state_sync.schema:
+        environments_table = f"{state_sync.schema}.{environments_table}"
+
+    alter_table_exp = exp.AlterTable(
+        this=exp.to_table(environments_table),
+        actions=[
+            exp.ColumnDef(
+                this=exp.to_column("suffix_target"),
+                kind=exp.DataType.build("text"),
+            )
+        ],
+    )
+    engine_adapter.execute(alter_table_exp)
+
+    environments = []
+    columns = [
```

Comment:
why are you enumerating the columns? just do a select *

Reply:
I prefer always being explicit.